client/admin-pointers.js

```diff
@@ -22,17 +20,29 @@ jQuery( document ).ready( function( $ ) {
 			return;
 		}
 
+		const target = $( pointer.target );
+
 		const options = $.extend( pointer.options, {
 			close: function() {
 				$.post( ajaxurl, {
 					pointer: pointer.id,
 					action: 'dismiss-wp-pointer',
 				} );
+
+				if ( pointer.dim ) {
+					$( '#wcs-pointer-page-dimmer' ).fadeOut( 500 );
+				}
+
 				show_pointer( pointers, i + 1 );
 			},
 		} );
 
-		$( pointer.target ).pointer( options ).pointer( 'open' );
+		target.pointer( options ).pointer( 'open' );
+		if ( pointer.dim ) {
+			target.css( 'z-index', 9999 );
+			$( 'body' ).append( '<div id="wcs-pointer-page-dimmer" class="wcs-pointer-page-dimmer"></div>' );
```

Comment:
I added the overlay to the footer if we invoke any pointers. It feels like it executes differently, but it seems to be working. Let me know if you notice any issues with it.

Not sure I understand the thing about scrolling. I don't think disabling it is a very good idea. On a smaller screen the user might actually want to scroll down to the metabox and it's pointer?